src/cache.ts

```diff
@@ -15,17 +15,17 @@ export default class Cache implements CacheInterface {
     return this.__cache.get(_key)
   }
 
-  set(key: keyInterface, value: any): any {
+  set(key: keyInterface, value: any) {
     const [_key] = this.serializeKey(key)
     this.__cache.set(_key, value)
     this.notify()
   }
 
-  keys() {
+  keys(): string[] {
     return Array.from(this.__cache.keys())
   }
 
-  has(key: keyInterface) {
+  has(key: keyInterface): boolean {
```

Comment:
this could be inferred since this.__cache is a Map

Reply:
@promer94 I guess `has` would be safe to return boolean type?

Reply:
https://github.com/vercel/swr/blob/651c368833164945a8ee5708eb3dd6091241f05b/src/types.ts#L124-L133

i just find we already had the interface of cache.  so we may don't need to add return type for these functions.